rcc.pri

```diff
@@ -22,11 +25,5 @@ CONFIG(release, debug|release) {
 
 load(resources)
 
-win32 {
-	WINPWD = $$shell_path($${PWD})
-	RCC_DEPENDS = $$escape_expand(\")$${WINPWD}\\scripts\\rcc-depends.py$$escape_expand(\")
-	rcc.depend_command = python $${RCC_DEPENDS} ${QMAKE_FILE_IN}
-} else {
-	RCC_DEPENDS = $$escape_expand(\")$${PWD}/scripts/rcc-depends.py$$escape_expand(\")
-	rcc.depend_command = $${RCC_DEPENDS} ${QMAKE_FILE_IN}
-}
+RCC_DEPENDS = $$escape_expand(\")$${PWD}/scripts/rcc-depends.py$$escape_expand(\")

```

Comment:
This seems wrong. On Windows $PWD will be with backslashes, so you're construction a path c:\that\looks\like/this/scripts/rcc-depends.py